### PR TITLE
Fix time value based on UTC to PST time zones

### PR DESCRIPTION
**Pull request by CodingVelocista**

### CODE REVIEW

File: embc-app/ViewModels/PrintReferral.cs [hunk 1]
```diff
@@ -91,12 +91,13 @@ private string MapPrintEvacueeTypeCode(Models.Db.Evacuee evacuee)
 
     public class PrintReferral : Referral
     {
+        private readonly TimeZoneInfo pst = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
```
Comment (ytqsl) [inline embc-app/ViewModels/PrintReferral.cs]: Will this handle PDT and PST?

Comment (CodingVelocista) [reply embc-app/ViewModels/PrintReferral.cs]: Good point, I know how to ensure PDT, let me add it

Comment (CodingVelocista) [reply embc-app/ViewModels/PrintReferral.cs]: Looks like the time PST time zone supports Daylight savings into the conversion.  So nothing has to be done for now.  If we move to a parameterized timezone, we'll have to consider it based on the timezone selected.

File: embc-app/ViewModels/PrintReferral.cs [hunk 2]
```diff
@@ -122,6 +122,14 @@ private string ConvertCarriageReturnToHtml(string value)
         {
             return value?.Replace("\n", "<br />")?.Replace("\r", "<br />");
         }
+
+        private string GetLocalTime(DateTime dateTime)
+        {
+            var pst = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
```
Comment (ytqsl) [inline embc-app/ViewModels/PrintReferral.cs]: Will this handle PDT conversion as well?

Comment (CodingVelocista) [reply embc-app/ViewModels/PrintReferral.cs]: Yes, it will.

Comment (CodingVelocista) [reply embc-app/ViewModels/PrintReferral.cs]: Each timezone either supports or doesn't support PDT conversion.

Comment (CodingVelocista) [reply embc-app/ViewModels/PrintReferral.cs]: PST does

**Pull request merged**